apps/next-next-app/src/pages/_app.tsx

```diff
@@ -0,0 +1,10 @@
+import '../styles/globals.css'
+import React from "react"
```

Comment:
Note, always import 3rd party packages first. And this react import is not necessary above React 18. Move the styles import below and pad them with a new line.